src/hooks/screen/useSignedChatInfoHook.ts

```diff
@@ -2,37 +2,38 @@ import * as React from 'react';
 import {useNavigation} from '@react-navigation/core';
 
 import ChannelList from '../../database/schema/ChannelListSchema';
-import UseAnonymousChatInfoScreenHook from '../../../types/hooks/screens/useAnonymousChatInfoScreenHook.types';
+import UseSignedChatInfoScreenHook from '../../../types/hooks/screens/useSignedChatInfoScreenHook.types';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import {ChannelListMemberSchema} from '../../../types/database/schema/ChannelList.types';
 import {getAnonymousUserId} from '../../utils/users';
 import {getUserId} from '../../utils/token';
 
-function useAnonymousChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
+function useSignedChatInfoScreenHook(): UseSignedChatInfoScreenHook {
   const {localDb, channelList} = useLocalDatabaseHook();
   const {selectedChannel, goBack} = useChatUtilsHook();
-
   const navigation = useNavigation();
 
   const [channelInfo, setChannelInfo] = React.useState(null);
 
-  const initChatInfoData = async () => {
+  const initializeChatInfoData = async () => {
     if (!localDb) return;
     const myId = await getUserId();
+
     const myAnonymousId = await getAnonymousUserId();
     const data = await ChannelList.getChannelInfo(
       localDb,
       selectedChannel?.id,
       myId,
       myAnonymousId
     );
+
     setChannelInfo(data);
   };
 
   const onContactPressed = (item: ChannelListMemberSchema) => {
     if (!item?.user?.isMe)
-      navigation.push('OtherProfile', {
+      navigation.navigate('OtherProfile', {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The navigation method has been changed from `push` to `navigate`. This could potentially change the behavior of the application. If the intention is to keep the user on the same stack and not create a new one, then `navigate` is the correct method to use. However, if the intention is to allow the user to go back to the previous screen with the back button, then `push` should be used.



<!-- This is an auto-generated comment by OSS CodeRabbit -->